### PR TITLE
Moved Validators into the package and wrote README.md

### PR DESCRIPTION
**Pull request by gigbat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
...
+* Registration of a new user.
+* View a list of available footballs.
+* View the list of football halls.
+* View the list of footballs.
+* Make order.
+* View the shopping cart.
+####Authorized admin:
+* Registration of a new user/admin.
+* View/Add the list of football halls.
+* View/Add/Update/Remove the list of footballs.
```
Comment (Alex-Korzun) [inline README.md]: What do you mean by list of footballs? Maybe football matches?

File: README.md [hunk 2]
```diff
@@ -1 +1,44 @@
...
+####Authorized user:
+* Registration of a new user.
+* View a list of available footballs.
+* View the list of football halls.
+* View the list of footballs.
+* Make order.
+* View the shopping cart.
+####Authorized admin:
+* Registration of a new user/admin.
+* View/Add the list of football halls.
```
Comment (Alex-Korzun) [inline README.md]: What is football hall? Maybe stadium is better?)

File: src/main/java/com/dev/football/config/SecurityConfig.java [hunk 3]
```diff
@@ -28,12 +28,12 @@ public void configureGlobal(AuthenticationManagerBuilder auth) throws Exception
...
-                .antMatchers(HttpMethod.POST,"/movies/**").hasRole("ADMIN")
-                .antMatchers(HttpMethod.GET,"/movies/**").hasRole("ADMIN")
-                .antMatchers(HttpMethod.POST,"/cinema-halls/**").hasRole("ADMIN")
-                .antMatchers(HttpMethod.GET,"/cinema-halls/**").hasRole("ADMIN")
-                .antMatchers(HttpMethod.POST,"/movie-sessions/**").hasRole("ADMIN")
-                .antMatchers(HttpMethod.GET,"/movie-sessions/**").hasRole("ADMIN")
+                .antMatchers(HttpMethod.POST,"/footballs/**").hasRole("ADMIN")
+                .antMatchers(HttpMethod.GET,"/footballs/**").hasRole("ADMIN")
+                .antMatchers(HttpMethod.POST,"/football-halls/**").hasRole("ADMIN")
+                .antMatchers(HttpMethod.GET,"/football-halls/**").hasRole("ADMIN")
```
Comment (Alex-Korzun) [inline src/main/java/com/dev/football/config/SecurityConfig.java]: I'd recommend you to change `footbal` to `football match` or just `match`. And change it every where. But it's up to you. And `footbal-halls` to `football-stadiums`.

**Review by Alex-Korzun — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gigbat) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (gigbat) [reply README.md]: yes)

File: src/main/java/com/dev/football/config/SecurityConfig.java (hunk 3 above)
Comment (gigbat) [reply src/main/java/com/dev/football/config/SecurityConfig.java]: done

File: README.md [hunk 4]
```diff
@@ -1 +1,44 @@
-#CINEMA
+#Football-tickets
```
Comment (RomanDubovskyi) [inline README.md]: It's up to you, but I would advise using a different template for the readme. Not it looks similar to other projects visually.

**Review by RomanDubovskyi — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (gigbat) [reply README.md]: okey

**Pull request merged**